spec/lib/given/extensions_spec.rb

```diff
@@ -180,6 +180,16 @@
     Then { result == :ok }
   end
 
+  if RSpec::Version::STRING.start_with? "2"
```

Comment:
Hey @searls it's been a while but I bet it was really intended to distinguish between RSpec 2 & 3.  I have no idea whether RSpec 1 would work.  Arguably best would have been to do feature testing rather than version testing -- i.e. test on whether `RSpec.current_example` is defined, with a comment that it was introdocued in RSpec 3.  (If it's defined at all in the outer context of the specs.)